can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -158,15 +158,23 @@ def __init__(self, channel, can_filters=None, **kwargs):
         with open_lock:
             ics.open_device(self.dev)
 
-        if "bitrate" in kwargs:
-            for channel in self.channels:
-                ics.set_bit_rate(self.dev, kwargs.get("bitrate"), channel)
-
-        fd = kwargs.get("fd", False)
-        if fd:
-            if "data_bitrate" in kwargs:
+        try:
+            if "bitrate" in kwargs:
                 for channel in self.channels:
-                    ics.set_fd_bit_rate(self.dev, kwargs.get("data_bitrate"), channel)
+                    ics.set_bit_rate(self.dev, kwargs.get("bitrate"), channel)
+
+            fd = kwargs.get("fd", False)
```

Comment:
Could be inlined into the if statement's condition. (Does not need to be a variable.) That's nothing too important though. ;)

Reply:
Agree, but will not touch it in this PR